.github/exemptions.md

```diff
@@ -0,0 +1,31 @@
+# Reasons for ignoring certain checks
+
+## Megalinter Disabled Linters
+
+- jscpy - CopyPaste scanning is disabled
+- JAVASCRIPT_STANDARD - We use eslint instead
+
+## Specific linter exemptions
+
+- cspell - "ignoreRegExpList": "/[0-9a-f]{8}.js/gi" - scripts/downlaod.js will save files with this name
+- cspell - "ignoreRegExpList": "/[0-9a-f]{8}\\_ABI.json/gi" - scripts/downlaod.js will save files with this name
```

Comment:
```suggestion
- cspell - "ignoreRegExpList": "/[0-9a-f]{8}\\_ABI.json/gi" - scripts/download.js will save files with this name
```

Reply:
addressed in e1250bd6ccf32489638864a367c380f8b73589cc